Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4381,9 +4440,62 @@ ObservableList<ControlProperty> IWindowExplorer.GetElementProperties(ElementInfo
                 ElementInfo.ElementObject = el;
             }
 
-            //Learn more properties
+            //Base properties 
+            list.Add(new ControlProperty() { Name = "Platform Element Type", Value = ElementInfo.ElementType });
+            list.Add(new ControlProperty() { Name = "Parent IFrame", Value = ElementInfo.Path });
+            list.Add(new ControlProperty() { Name = "XPath", Value = ElementInfo.XPath });
+            list.Add(new ControlProperty() { Name = "Relative XPath", Value = ((HTMLElementInfo)ElementInfo).RelXpath });
+            list.Add(new ControlProperty() { Name = "Height", Value = ((IWebElement)ElementInfo.ElementObject).Size.Height.ToString() });
+            list.Add(new ControlProperty() { Name = "Width", Value = ((IWebElement)ElementInfo.ElementObject).Size.Width.ToString() });
+            list.Add(new ControlProperty() { Name = "X", Value = ((IWebElement)ElementInfo.ElementObject).Location.X.ToString() });
+            list.Add(new ControlProperty() { Name = "Y", Value = ((IWebElement)ElementInfo.ElementObject).Location.Y.ToString() });
+            list.Add(new ControlProperty() { Name = "Value", Value = ElementInfo.Value });
+
 
-            if (el != null)
+            if (((HTMLElementInfo)ElementInfo).HTMLElementObject != null)
+            {
+                if (ElementInfo.IsElementTypeSupportingOptionalValues(ElementInfo.ElementTypeEnum))
+                {
+
+                    foreach (HtmlNode childNode in ((HTMLElementInfo)ElementInfo).HTMLElementObject.ChildNodes)
+                    {
+                        if (!childNode.Name.StartsWith("#") && !string.IsNullOrEmpty(childNode.InnerText))
+                        {
+                            string[] tempOpVals = childNode.InnerText.Split('\n');
+                            foreach (string cuVal in tempOpVals)
+                            {
+                                ElementInfo.OptionalValuesObjectsList.Add(new OptionalValue() { Value = cuVal, IsDefault = false });
+                            }
+
+                        }
+
+                    }
+
+                    if (ElementInfo.OptionalValuesObjectsList.Count > 0)
+                    {
+                        ElementInfo.OptionalValuesObjectsList[0].IsDefault = true;
+                    }
+                    list.Add(new ControlProperty() { Name = "Optional Values", Value = ElementInfo.OptionalValuesObjectsListAsString.Replace("*", "") });
```

Comment:
please enter the "list.add.." into the if, it should be added only if list is bigger than zero.
this is also the conflict you have with latest after i pushed POM Delta changes

Reply:
Corrected